.github/workflows/windows-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: e2e tests on Windows
+
+on:
+  schedule:
+    - cron: '13 5 * * 1-5'
+  workflow_dispatch: {}
+defaults:
+  run:
+    shell: powershell
+jobs:
+
+  e2e-tests:
+    timeout-minutes: 90
+    runs-on: [self-hosted, Windows, X64]
+    steps:
+      - run: wsl --unregister rancher-desktop
+        continue-on-error: true
+      - run: wsl --unregister rancher-desktop-data
+        continue-on-error: true
+      - run: >
```

Comment:
I don't understand the reason for this > symbol. The commands can be written in the same line of `run`

Reply:
That's just wrapping because the line was long. (`>` means all new lines are replaced by a space; see [specs](https://yaml.org/spec/1.2.2/#rule-b-l-folded).). I can move it up, though, since it ended up short enough.